src/components/Answers/Answers.tsx

```diff
@@ -21,12 +21,14 @@ interface Iprops{
   } 
 }
 
-const Answers: React.FC<Iprops> = ({ currentCountry, questions, guess }) => {
-
-  const [answer, setAnswer] = useState<string | number>('')
-  const [currency] = currentCountry.currencies
-  const [languages, setLanguage] = useState<string[]>()
+const Answers: React.FC<IAnswerProps> = ({ currentCountry, questions, guess }) => {
+  const [answer, setAnswer] = useState<string | number>('');
+  const [currency] = currentCountry.currencies;
+  const [languages, setLanguage] = useState<string[]>();
 
+   // *******************************************************
```

Comment:
Love the additions of the labels to explain the functionality. This is a great step in showing developer empathy!